### ISSUE TITLE
Explore the lengths of the noisy dataset and test dataset

### ISSUE THREAD

**Issue by JoshVarty** [JoshVarty/AudioTagging#25]
We should take a look at the `noisy` and `test` datasets in our exploratory data analysis. [My understanding](https://www.kaggle.com/c/freesound-audio-tagging-2019/data) is that the `test` dataset is from the same source as the `curated` dataset:

>The test set is used for system evaluation and consists of manually-labeled data from FSD. Since most of the train data come from YFCC, some acoustic domain mismatch between the train and test set can be expected. All the acoustic material present in the test set is labeled, except human error, considering the vocabulary of 80 classes used in the competition.

Are the clips in our `test` set the same length as the ones in the `curated` set?

**Comment by JoshVarty** [comment]
### Curated Training Set
![image](https://user-images.githubusercontent.com/1249087/57097255-502dcc00-6ce5-11e9-9dab-971a9788351b.png)

### Noisy Training Set

![image](https://user-images.githubusercontent.com/1249087/57097268-59b73400-6ce5-11e9-9121-8feec1525a7a.png)

Almost all clips are 15 seconds long.

### Test Set

![image](https://user-images.githubusercontent.com/1249087/57097290-63d93280-6ce5-11e9-8e4f-587a9eacd772.png)


So it looks to me like the test set is taken from roughly the same distribution as the curated training set.
